hypothesis-python/tests/conjecture/common.py

```diff
@@ -447,27 +450,27 @@ def ir(*values: list[ChoiceT]) -> list[ChoiceNode]:
     }
     nodes = []
     for value in values:
-        override_kwargs = {}
+        override_constraints = {}
         if isinstance(value, tuple):
-            (value, override_kwargs) = value
-            if override_kwargs is None:
-                override_kwargs = {}
+            (value, override_constraints) = value
+            if override_constraints is None:
+                override_constraints = {}
 
-        (choice_type, kwargs) = mapping[type(value)]
+        (choice_type, constraints) = mapping[type(value)]
 
         nodes.append(
             ChoiceNode(
                 type=choice_type,
                 value=value,
-                kwargs=kwargs | override_kwargs,
+                constraints=constraints | override_constraints,
                 was_forced=False,
             )
         )
 
     return tuple(nodes)
 
 
-def float_kw(
+def float_constr(
```

Comment:
I don't love `_constr` as a suffix, but don't have a better suggestion - `make_x_constraint()` seems too verbose, but using `_constraint` is too similar to the strategy names, so just noting in case anyone else has suggestions I guess.  Maybe even keeping the old name?  (`constraints=float_kw()` seems basically fine to me...)